### PR TITLE
SRCH-145

### PR DESCRIPTION
**Pull request by juliacollins**

### CODE REVIEW

File: lib/search_solr_tools/translators/nsidc_json.rb [hunk 1]
```diff
@@ -75,9 +75,10 @@ def translate_sensor_to_facet_sensor(json)
         return facet_values if json.nil?
 
         json.each do |json_entry|
+          long_name = json_entry['shortName'].eql?(json_entry['longName']) ? '' : json_entry['longName']
           sensor_bin = Helpers::SolrFormat.facet_binning('sensor', json_entry['shortName'].to_s)
           facet_values << if sensor_bin.eql? json_entry['shortName']
-                            "#{json_entry['longName']} | #{json_entry['shortName']}"
+                            "#{long_name} | #{json_entry['shortName']}"
```
Comment (scott-lewis-nsidc) [inline lib/search_solr_tools/translators/nsidc_json.rb:81]: If the long_name is blank (due to the change above) that would make this line look like ` | short_name`.  Is this the desired result?  Or would it be better to not have the divider in there?

File: search_solr_tools.gemspec [hunk 2]
```diff
@@ -8,7 +8,8 @@ Gem::Specification.new do |spec|
   spec.name          = 'search_solr_tools'
   spec.version       = SearchSolrTools::VERSION
   spec.authors       = ['Chris Chalstrom', 'Michael Brandt', 'Jonathan Kovarik', 'Luis Lopez', 'Stuart Reed', 'Julia Collins', 'Scott Lewis']
-  spec.email         = ['cchalstr@nsidc.org', 'mbrandt@colorado.edu', 'kovarik@nsidc.org', 'luis.lopezespinosa@colorado.edu', 'stuart.reed@colorado.edu', 'jcollins@nsidc.org', 'scott.lewis@nsidc.org']
+  spec.email         = ['cchalstr@nsidc.org', 'mbrandt@colorado.edu', 'Jonathan.Kovarik@colorado.edu', 'luis.lopezespinosa@colorado.edu', 'stuart.reed@colorado
```
Comment (scott-lewis-nsidc) [inline search_solr_tools.gemspec:11]: A few of these email addresses (Chris, Michael, and Stuart) aren't really valid anymore.  Obviously we still want to credit them, but do we perhaps want to remove their emails?

**Review by scott-lewis-nsidc — CHANGES_REQUESTED** [review]
Two minor changes suggested (though if both are ok as is, just open a new request and I'll approve).

File: lib/search_solr_tools/translators/nsidc_json.rb (hunk 1 above)
Comment (juliacollins) [reply lib/search_solr_tools/translators/nsidc_json.rb:81]: That is exactly the desired result. It's what the content looks like now.

File: search_solr_tools.gemspec (hunk 2 above)
Comment (juliacollins) [reply search_solr_tools.gemspec:11]: I'll remove the emails as you suggest. Thanks!

**Review by scott-lewis-nsidc — APPROVED** [review]

**Pull request merged**